Tests/HummingbirdTests/TracingTests.swift

```diff
@@ -378,8 +385,9 @@ final class TracingTests: XCTestCase {
         let router = HBRouterBuilder(context: HBTestRouterContext.self)
         router.middlewares.add(SpanMiddleware())
         router.middlewares.add(HBTracingMiddleware())
-        router.get("/") { _, context -> EventLoopFuture<HTTPResponseStatus> in
-            return context.eventLoop.scheduleTask(in: .milliseconds(2)) { return .ok }.futureResult
+        router.get("/") { _, context -> HTTPResponseStatus in
+            return .ok
+            try await Task.sleep(for: .milliseconds(2))
```

Comment:
Did you mean to have the sleep after the return

Reply:
Oops, that's a merge mistake